Resources/Prototypes/Roles/Jobs/departments.yml

```diff
@@ -104,18 +109,20 @@
   - Chaplain # DeltaV - Move Chaplain into Epistemics
   - ForensicMantis # Nyanotrasen - ForensicMantis, see Resources/Prototypes/Nyanotrasen/Roles/Jobs/Epistemics/forensicmantis.yml
 
-- type: department
-  id: Specific
-  description: department-Specific-description
-  color: "#9FED58"
-  weight: 10
-  roles:
-  - Boxer
-  - Reporter
-  - Zookeeper
-  - Psychologist
-  - MartialArtist # Nyanotrasen - MartialArtist, see Resources/Prototypes/Nyanotrasen/Roles/Jobs/Wildcards/martialartist.yml
-  - Gladiator # Nyanotrasen - Gladiator, see Resources/Prototypes/Nyanotrasen/Roles/Jobs/Wildcards/gladiator.yml
-  - Prisoner # Nyanotrasen - Prisoner, see Resrouces/Prototypes/Nyanotrasen/Roles/Jobs/Wildcards/prisoner.yml
-  - Brigmedic # DeltaV - Corpsman, see Resources/Prototypes/DeltaV/Roles/Jobs/Security/brigmedic.yml
-  primary: false
+# Parkstation-Oligarchy Start
+#- type: department
+#  id: Specific
```

Comment:
What's this? DeltaV (or Wizden) delegates their less important roles to a weird fake Department? Lame as hell. Move Psych into Medical and Reporter into Service, please.

Reply:
uh, it uh, Itss already like that.

Reply:
It's possible I was looking at an old commit then? I believe the "department" wasn't commented out when I left that, but I may have had a commit filter on by mistake.